### ISSUE TITLE
HID support

### ISSUE THREAD

**Issue by acai66** [kimocoder/nethunter_kernel_op6#1]
HID not needed anymore, all can be done in init.nethunter.rc

**Comment by kimocoder** [comment]
Yeah I know about that one, just haven't utilized it yet. It's not my 'yard', rather re4son's garden :dancer:

**Comment by Owningle** [comment]
> HID not needed anymore, all can be done in init.nethunter.rc
Is it possible to use apps that use the other hid method / patch using this?
